packages/dart/lib/src/network/parse_dio_client.dart

```diff
@@ -1,13 +1,11 @@
-import 'dart:io';
-
 import 'package:dio/dio.dart' as dio;
 import 'package:parse_server_sdk/parse_server_sdk.dart';
 
 import 'dio_adapter_io.dart' if (dart.library.js) 'dio_adapter_js.dart';
 
 class ParseDioClient extends ParseClient {
-  ParseDioClient(
-      {bool sendSessionId = false, SecurityContext? securityContext}) {
+  // securityContext is SecurityContext
+  ParseDioClient({bool sendSessionId = false, dynamic securityContext}) {
```

Comment:
Why the securityContext is dynamic?

what was wrong with `SecurityContext?` type

Reply:
I used dynamic to reduce memory in the library. It doesn't matter, if you think it should be returned, I will return it

Reply:
What about the conditional import:

```dart 
import 'dio_adapter_io.dart' if (dart.library.js) 'dio_adapter_js.dart';
```
When the library run on the web the `dio_adapter_js.dart` will be used. 

see this [answer](https://stackoverflow.com/a/75155651)

Reply:
I am currently using universal_io, do you think we should replace the above method?

Reply:
Even `universal_io` is still using `dart_io` in its imports for the web. take a look at [http_client.dart](https://github.com/dint-dev/universal_io/blob/3f23477b0976754a7c4e4976401316ae4055c585/lib/src/http_client.dart#LL45C13-L45C13) which in turn been exported in[ _exports_in_browser.dart](https://github.com/dint-dev/universal_io/blob/3f23477b0976754a7c4e4976401316ae4055c585/lib/src/_exports_in_browser.dart#LL18C3-L18C3)

the [exports](https://github.com/dint-dev/universal_io/blob/3f23477b0976754a7c4e4976401316ae4055c585/lib/io.dart#L24-L26)
```dart 
export 'src/_exports_in_vm.dart'
    if (dart.library.html) 'src/_exports_in_browser.dart'
    if (dart.library.js) 'src/_exports_in_nodejs.dart';
```
  
So keep it `dynamic` for now. until we find a proper way to get around this. because currently, the SDK is full of `dynamic` and `Object` types. that is not good for the long term.

Thanks for your work.